packages/validation/src/isArray.js

```diff
@@ -0,0 +1,3 @@
+export default function isArray(value) {
+    return Object.prototype.toString.call(value) === '[object Array]';
```

Comment:
this is fine but we have Array.isArray the it does the same :)

Reply:
I made this validator for 100% compatibity with ie... as Array.isArray is only supported in ie 9+

Reply:
but it can be removed